### DISCUSSION TITLE
寒流大佬，能不能开个系列，讲讲WPF的页面相关的

### DISCUSSION THREAD

**Discussion by MicroHeartWangZheng** [BYJRK/DotNet-Discussions#227] [WPF 开发]
本身是做服务端开发的，对前端Vue Css Html 还算熟悉，换工作发现纯用Net开发服务端的公司太少了，又不准备换语言，对Net 爱的深沉。你的视频几乎都看完了，讲的很好，每期都点赞、投币，对学习WPF很有帮助。 现在对WPF页面相关的知识感觉很困惑，写着感觉特别别扭，跟Css Html布局这些差别很大，本身做服务端的，感觉WPF的依赖属性、依赖注入、消息。。。这些很好理解。 就是这个WPF页面感觉超级难以理解。

**Comment by BYJRK** [comment]
这个主要还是要去熟悉WPF中的常用控件以及容器都有哪些，然后再去思考如何排版，以及控件的选择。可以看一些经典的案例（比如todo list，管理系统等）找找启发
